internal/eval/countminsketch_test.go

```diff
@@ -0,0 +1,318 @@
+				createCountMinSketch(key, opts, store)
+
+				source1 := "test"
+				opts, _ = newCountMinSketchOpts([]string{"1000", "5"})
+				createCountMinSketch(source1, opts, store)
+
+			},
+			input: []string{"cms_key5", "not_a_number", "test"},
+			output: []byte("-ERR cannot parse number for 'cms.merge' command\r\n"),
+		},
+		"more sources than specified": {
+			setup: func () {
+				key := "cms_key5"
+				opts, _ := newCountMinSketchOpts([]string{"1000", "5"})
+				createCountMinSketch(key, opts, store)
+
+				source1 := "test"
+				opts, _ = newCountMinSketchOpts([]string{"1000", "5"})
+				createCountMinSketch(source1, opts, store)
+
+				source2 := "test1"
+				opts, _ = newCountMinSketchOpts([]string{"1000", "5"})
+				createCountMinSketch(source2, opts, store)
+
+			},
+			input: []string{"cms_key5", "3", "test", "test1"},
+			output: []byte("-ERR invalid number of arguments to merge for 'cms.merge' command\r\n"),
+		},
+		"fewer sources than specified": {
+			setup: func () {
+				key := "cms_key5"
+				opts, _ := newCountMinSketchOpts([]string{"1000", "5"})
+				createCountMinSketch(key, opts, store)
+
+				source1 := "test"
+				opts, _ = newCountMinSketchOpts([]string{"1000", "5"})
+				createCountMinSketch(source1, opts, store)
+
+				source2 := "test1"
+				opts, _ = newCountMinSketchOpts([]string{"1000", "5"})
+				createCountMinSketch(source2, opts, store)
+
+			},
+			input: []string{"cms_key5", "1", "test", "test1"},
+			output: []byte("-ERR invalid number of arguments to merge for 'cms.merge' command\r\n"),
+		},
+		"missing weights": {
+			setup: func () {
+				key := "cms_key5"
+				opts, _ := newCountMinSketchOpts([]string{"1000", "5"})
+				createCountMinSketch(key, opts, store)
+
+				source1 := "test"
+				opts, _ = newCountMinSketchOpts([]string{"1000", "5"})
+				createCountMinSketch(source1, opts, store)
+
+			},
+			input: []string{"cms_key5", "1", "test", "WEIGHTS"},
+			output: []byte("-ERR invalid number of arguments to merge for 'cms.merge' command\r\n"),
+		},
+		"more weights than needed": {
+			setup: func () {
+				key := "cms_key5"
+				opts, _ := newCountMinSketchOpts([]string{"1000", "5"})
+				createCountMinSketch(key, opts, store)
+
+				source1 := "test"
+				opts, _ = newCountMinSketchOpts([]string{"1000", "5"})
+				createCountMinSketch(source1, opts, store)
+
+			},
+			input: []string{"cms_key5", "1", "test", "WEIGHTS", "1", "2"},
+			output: []byte("-ERR invalid number of arguments to merge for 'cms.merge' command\r\n"),
+		},
+		"correct case": {
+			setup: func () {
+				key := "cms_key5"
+				opts, _ := newCountMinSketchOpts([]string{"1000", "5"})
+				createCountMinSketch(key, opts, store)
+
+				source1 := "test"
+				opts, _ = newCountMinSketchOpts([]string{"1000", "5"})
+				createCountMinSketch(source1, opts, store)
+
+			},
+			input: []string{"cms_key5", "1", "test"},
+			output: clientio.RespOK,
+		},
+		"correct case with given weights": {
```

Comment:
for ease of debugging in CI logs, these test case names should include the word CMS.